src/components/Dashboard/TodoList/PaymentPendingOnlineBanking.vue

```diff
@@ -30,16 +36,25 @@
 
 <script lang="ts">
 import { Component, Prop, Vue } from 'vue-property-decorator'
+import { PayServices } from '@/services/'
 
 @Component({})
 export default class PaymentPendingOnlineBanking extends Vue {
   /** The subject filing. */
   @Prop({ required: true }) readonly filing!: any
+  @Prop({ required: true }) readonly getPayApiUrl!: string
+
+  cfsAccountId: string | null = null;
 
   /** The draft title of the subject filing. */
   get draftTitle (): string {
     return this.filing?.draftTitle || 'filing'
   }
+
+  async mounted () {
+    const accountId = JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id
```

Comment:
This already exists outside and should be passed in as a prop:

![image](https://github.com/bcgov/business-filings-ui/assets/3484109/947e7978-32b3-4233-a013-23d5ef57122e)